### ISSUE TITLE
Incompatibility with Kibana 5.5.0

### ISSUE THREAD

**Issue by fbaligand** [seadiaz/computed-columns#12]
Hi,

- I download : computed-columns-0.6.0-5.3.3.zip
- I modify `package.json` to set kibana version to 5.5.0
- I install the updated zip in Kibana 5.5.0 => it works fine : "Plugin installation complete"
- Then, I start Kibana => no error
- But when I access Kibana in my browser, I get the following `fatal` error :

```
 Fatal Error
TypeError: _modules2.default is undefined (http://localhost:5601/bundles/kibana.bundle.js?v=15382:279)

Version: 5.5.0
Build: 15382

Error: TypeError: _modules2.default is undefined (http://localhost:5601/bundles/kibana.bundle.js?v=15382:279)
[308]/window.onerror@http://localhost:5601/bundles/commons.bundle.js?v=15382:76:8876
```

**Comment by seadiaz** [comment]
Kibana 5.5.x is not officially supported, but I would try with computed-columns-0.6.0-5.4.3.zip instead.

I will remain attentive to any update from you.
--
Sebastián Díaz

**Comment by fbaligand** [comment]
My mistake : I would say computed-columns-0.6.0-5.4.3.zip

**Comment by fbaligand** [comment]
And thanks a lot for your work on this plugin !
I love this plugin !
And I plan to provide a enhancement soon...

**Comment by seadiaz** [comment]
Release 0.7.0 support Kibana 5.5.5

**Comment by fbaligand** [comment]
@seadiaz 
Sorry but Kibana 5.5.5 doesn't exist. 
Last 5.x version is : 5.5.0

**Comment by seadiaz** [comment]
You have right.
I fixed version to 5.5.0.

:D


--
Sebastián Díaz
Skype: sebadiaz
seadiaz@gmail.com

**Comment by fbaligand** [comment]
Great !
Nothing important, but I think there is still a typo in `README.md` :smiley:
